cartography/intel/aws/ssm.py

```diff
@@ -0,0 +1,192 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import aws_handle_regions
+from cartography.util import dict_date_to_epoch
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_instance_ids(neo4j_session: neo4j.Session, region: str, current_aws_account_id: str) -> List[str]:
+    get_instances_query = """
+    MATCH (:AWSAccount{id: {AWS_ACCOUNT_ID}})-[:RESOURCE]->(i:EC2Instance)
+    WHERE i.region = {Region}
+    RETURN i.id
```

Comment:
We may also need to paginate this query.

Reply:
I can't find a really simple way to do this, and all the pagination support in cartography itself is for writing, not querying. I think even with a very large number of results (100k+, which would be a very very large number of instances for a single region and one account), the query will be fast without pagination.

I'd happy to paginate it, but right now the only pagination support via cartography is for write queries. Maybe this is something we can followup, with library functions for doing queries like this?

Reply:
I should have linked, and you're right we don't currently do pagination for neo4j queries. But it can be done with SKIP and LIMIT https://neo4j.com/docs/cypher-manual/current/clauses/skip/. 

But I won't block the PR on this, since even 100k+ results can return very quickly.